src/btree/bt_sync.c

```diff
@@ -325,6 +325,16 @@ __wt_sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
         btree->syncing = WT_BTREE_SYNC_RUNNING;
         is_las = F_ISSET(btree, WT_BTREE_LOOKASIDE);
 
+        /*
+         * Add in lookaside reconciliation for standard files.
+         *
+         * FIXME-PM-1521: Remove the WT_BTREE_LOOKASIDE check, and assert that no updates from
+         * lookaside are copied to lookaside recursively.
+         */
+        rec_flags = WT_REC_CHECKPOINT;
+        if (!WT_IS_LOOKASIDE(btree) && !WT_IS_METADATA(btree->dhandle))
+            rec_flags |= WT_REC_LOOKASIDE;
```

Comment:
The `is_las` assignment could use `WT_IS_LOOKASIDE`.

Additionally, can you review & cleanup explicit checks of `WT_BTREE_LOOKASIDE` in other parts of the tree?

Reply:
Did it for all checks except one as it was more readable to leave it for that one check.